docs/installing-solidity.rst

```diff
@@ -230,6 +230,7 @@ Or, on Windows:
 Command-Line Build
 ------------------
 
+**Be sure to install External Dependencies before build.**
```

Comment:
Can you add `External Dependencies (see above)`? And an empty line between this and the next.